src/System.Devices.Gpio/System.Devices.Gpio/System.Devices.Gpio.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\..\tools\common.props" />
+  <PropertyGroup>
+    <Description></Description>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PackageTags>Command line parsing support corefxlab</PackageTags>
+    <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="System\**\*.*" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="UnixDriver.cs" />
```

Comment:
You shouldn't need to explicitly include all the files to compile. File globing should automatically grab everything within the System.Devices.Gpio folder.

Reply:
Thanks, I didn't do it myself. I think VS did it for me. It should be fixed in the next commit.